src/components/Utils/orderResource.js

```diff
@@ -17,26 +15,33 @@ const saveOrder = (order, mutator) => {
   return method(order);
 };
 
+export const getFullOrderNumber = ({ poNumberPrefix = '', poNumberSuffix = '', poNumber = '' } = {}) => (
```

Comment:
**FYI, not a required change**
usually brackets are needed if you return a react element. In this very case it should be OK without them.